lib/plugins/features/movies/controller.js

```diff
@@ -8,3 +8,16 @@ exports.create = (payload) => {
     return new Movie({ id: movie.id }).fetch();
   });
 };
+
+exports.list = (filter) => {
+  return new Movie().query((qb) => {
+    if (filter.release_year) {
+      qb.where('release_year', filter.release_year);
+    }
+
+    if (filter.title) {
+      qb.whereRaw('lower(title) = ?', filter.title);
+    }
+  })
```

Comment:
Just to give you some context, typically we group filtering logic into the model instead of the controller. Since the logic here is pretty small and is already tested, this is fine. Usually it is when the filtering starts to get more [complicated](https://github.com/lob/lob-api/blob/75e38a8303f010273f6acb17562fd04a15e8ac3c/src/models/postcard.js#L96-L152) and we start to need to reuse certain filters that it starts to make sense why it is a better fit in the model.